android_64/Dockerfile

```diff
@@ -0,0 +1,152 @@
+RUN update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 800 --slave /usr/bin/g++ g++ /usr/bin/g++-8
+
+ARG user=jenkins
+ARG group=jenkins
+ARG uid=1000 
+ARG gid=1000 
+ARG JENKINS_AGENT_HOME=/home/${user} 
+ENV JENKINS_AGENT_HOME ${JENKINS_AGENT_HOME} 
+RUN groupadd -g ${gid} ${group} \
+    && useradd -d "${JENKINS_AGENT_HOME}" -u "${uid}" -g "${gid}" -m -s /bin/bash "${user}"
+
+RUN echo "jenkins:qwerty123" | chpasswd
+
+VOLUME "/tmp" 
+
+ENV QT_VERSION=5.10.1 ANDROID_COMPILER=android_armv7
+
+RUN mkdir /var/run/sshd && \
+    mkdir $JENKINS_AGENT_HOME/Qt && \
+    chmod 777 -R $JENKINS_AGENT_HOME/Qt && \
+    mkdir $JENKINS_AGENT_HOME/workspace && \
+    chmod 777 -R $JENKINS_AGENT_HOME/workspace
+
+# Download QT installer
+RUN cd / && wget http://download.qt.io/official_releases/online_installers/qt-unified-linux-x64-online.run
+COPY qt-installer-noninteractive.qs /qt-installer-noninteractive.qs
+
+# Install QT
+RUN chmod a+x qt-unified-linux-x64-online.run
+RUN ./qt-unified-linux-x64-online.run --platform minimal --verbose --script qt-installer-noninteractive.qs
+
+EXPOSE 22
+
+ENV QTDIR "${JENKINS_AGENT_HOME}/Qt/${QT_VERSION}/${ANDROID_COMPILER}"
+ENV PATH $QTDIR/bin:"${JENKINS_AGENT_HOME}/Qt/Tools/QtCreator/bin":$PATH
+
+COPY ./clang-format /usr/local/bin
+
+WORKDIR $JENKINS_AGENT_HOME
+
+CMD ["/usr/sbin/sshd", "-D"]
+
+# Android
+# Install Dependencies
+RUN dpkg --add-architecture i386 && \
+    apt-get update && \
+    apt-get install -y libncurses5:i386 libstdc++6:i386 zlib1g:i386
+
+# Install java8
+RUN apt-get update && apt-get install -y software-properties-common && \
+    add-apt-repository -y ppa:webupd8team/java && apt-get update
+RUN echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections
+RUN apt-get install -y oracle-java8-installer && apt-get clean
+
+# Java Environment
+ENV JAVA8_HOME=/usr/lib/jvm/java-8-oracle JAVA_HOME=$JAVA8_HOME PATH=$PATH:$JAVA_HOME/bin
+
+# Download Android SDK tools
+RUN wget -q "http://dl.google.com/android/android-sdk_r24.4.1-linux.tgz" -O android-sdk.tgz && \
+    tar -xzf android-sdk.tgz -C /opt && \
+    rm android-sdk.tgz
+
+# Android SDK Environment
+ENV ANDROID_HOME=/opt/android-sdk-linux PATH=$PATH:$ANDROID_HOME/tools PATH=$PATH:$ANDROID_HOME/tools/bin \
+    PATH=$PATH:$ANDROID_HOME/platform-tools PATH=$PATH:$ANDROID_HOME/build-tools/24.0.3
+
+# Install Android SDK components
+RUN echo y | android update sdk --no-ui --all --filter \
+    "platform-tools,build-tools-28.0.2,android-22,android-24,android-25,android-26,android-27" && \
+    echo y | android update sdk --no-ui --all --filter \
+    "extra-android-m2repository,extra-google-m2repository,extra-android-support"
+
+# Install Android NDK
+RUN cd /opt && wget --output-document=android-ndk.zip \
+    --quiet http://dl.google.com/android/repository/android-ndk-r17b-linux-x86_64.zip && \
```

Comment:
can you test replacing this with http://dl.google.com/android/repository/android-ndk-r17c-linux-x86_64.zip ?
It's the latest version for r17

Reply:
Please test it.